layouts/partials/hooks/body-end.html

```diff
@@ -34,10 +34,7 @@
         swal("Oh, no…","Sorry, your browser doesn't support copying this example to your clipboard.");
         succeed = false;
     }
-    if (succeed) {
-      swal("Copied to clipboard: ",elem);
-      return succeed;
-    } else {
+    if (!succeed) {
```

Comment:
Instead how about flashing a "copied" message.
For e.g. when you click on the "Copy text" button in this example https://www.w3schools.com/howto/howto_js_copy_clipboard.asp, you'll get a nice "copied" message flashing.

Reply:
Having something like what Tim has mentioned in this comment https://github.com/kubernetes/website/issues/31258#issuecomment-1011040900, would be nice!